playbooks/common/openshift-cluster/redeploy-certificates/registry.yml

```diff
@@ -48,10 +48,6 @@
 
   # Replace dc/docker-registry certificate secret contents if set.
   - block:
-    - name: Load lib_openshift modules
-      include_role:
-        name: lib_openshift
-
```

Comment:
Why remove it here rather than making it static? Isn't it necessary for oc_service task below this?

Reply:
Nevermind, I see that the role is called at the top.